domain/src/main/kotlin/no/nav/su/se/bakover/domain/revurdering/Revurdering.kt

```diff
@@ -1092,3 +1141,21 @@ fun Revurdering.medFritekst(fritekstTilBrev: String) =
         is RevurderingTilAttestering.IngenEndring -> copy(fritekstTilBrev = fritekstTilBrev)
         is UnderkjentRevurdering.IngenEndring -> copy(fritekstTilBrev = fritekstTilBrev)
     }
+
+enum class Vurderingstatus(@JsonValue val status: String) {
```

Comment:
Disse kan vel flyttes ut og kanskje ha egne typer for serdes der vi trenger det eller bare mappe til/fra String